src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1256,102 +1258,103 @@
-  "advancements.minecolonies.complete_build_request_1_cowboy.description": "Build a Cowhand's Hut to bring in that beef, remember you'll need to bring the cows to your Cowhand for them to be usable!",
-  "advancements.minecolonies.complete_build_request_1_crusher.title": "I'm Crushed",
-  "advancements.minecolonies.complete_build_request_1_crusher.description": "Building a Crusher's Hut will enable you to get things like gravel from cobblestone!",
+  "advancements.minecolonies.complete_build_request_1_cook.description": "Build a Restaurant to give your citizens a place to get food, the Cook will cook meats and give food to you and your citizens.",
+  "advancements.minecolonies.complete_build_request_1_cowboy.title": "Yeeehaw!",
+  "advancements.minecolonies.complete_build_request_1_cowboy.description": "Build a Cowhand's Hut to bring in that beef. Remember to bring the cows to your Cowhand.",
+  "advancements.minecolonies.complete_build_request_1_crusher.title": "A Crushing Feeling",
+  "advancements.minecolonies.complete_build_request_1_crusher.description": "Build a Crusher's Hut to get things like gravel from cobblestone!",
   "advancements.minecolonies.complete_build_request_1_fisherman.title": "Something Smells Fishy",
   "advancements.minecolonies.complete_build_request_1_fisherman.description": "Build a Fisher's Hut close to water to bring in those deep sea morsels",
   "advancements.minecolonies.complete_build_request_1_library.title": "Higher Learning",
-  "advancements.minecolonies.complete_build_request_1_library.description": "If you want more skilled labor, you'd best build a Library to teach your citizens!",
+  "advancements.minecolonies.complete_build_request_1_library.description": "If you want more skilled labor, you'd best build a Library to educate your citizens",
   "advancements.minecolonies.complete_build_request_1_shepherd.title": "Mary had a Little Lamb",
-  "advancements.minecolonies.complete_build_request_1_shepherd.description": "Sometimes you really need some soft wool in your life. Build a Shepherd's Hut to take advantage of those poor unsuspecting sheep!",
+  "advancements.minecolonies.complete_build_request_1_shepherd.description": "Build a Shepherd's Hut to take advantage of those poor unsuspecting sheep",
   "advancements.minecolonies.complete_build_request_1_sifter.title": "Diamond in the Rough",
-  "advancements.minecolonies.complete_build_request_1_sifter.description": "Sometimes you'll find goodies inside random blocks. Build a Sifter's Hut to take advantage of that!",
-  "advancements.minecolonies.complete_build_request_1_swine_herder.title": "Getting the Bacon",
-  "advancements.minecolonies.complete_build_request_1_swine_herder.description": "Nobody can resist bacon, build a Swineherd's Hut so your citizens don't miss out!",
+  "advancements.minecolonies.complete_build_request_1_sifter.description": "Build a Sifter's Hut to take advantage of extra resources hiding in blocks",
```

Comment:
```suggestion
  "advancements.minecolonies.complete_build_request_1_sifter.description": "Build a Sifter's Hut to take advantage of extra resources hiding in blocks!",
```

Reply:
My point was to avoid the excessive use of exclamation marks! Because vanilla doesn't use them! And if everything's emphasized, effectively nothing is!